tos/tests/test_middleware.py

```diff
@@ -146,7 +147,7 @@ class Request(object):
             def is_ajax(self):
                 return True
 
-        mw = UserAgreementMiddleware()
+        mw = UserAgreementMiddleware(HttpResponse())
 
```

Comment:
Wondering if there's an easy way we can run a test on django < 4.0 where we don't pass an arg to this, to maintain backwards compat

Reply:
I could write the test twice, once with, once without passing the argument, and use the `@skipIf` decorator if the Django version is less / greater 4.0. https://docs.djangoproject.com/en/4.2/topics/testing/tools/#skipping-tests